README.md

```diff
@@ -1,12 +1,27 @@
-# quilt-example-mod
-Official Quilt example mod.
+# Quilt Example Mod
 
+The official Quilt example mod. You can use it as a template for you own mods!
 
-## To use this template
-1. Click on `Use this Template`
-2. Clone the resulted repo on your pc
-3. Enjoy!
+## Usage
+
+In order to use this mod as a template:
+
+1. Create a new repository from this template with `Use this Template`
+2. Clone the recently-created repo on your PC
+3. Make the necessary changes in order to make it yours:
+    - Update `gradle.properties` in order to use your Maven group and mod ID
+        - If you don't know which Maven group to use, and you are planning to host the mod's source code on GitHub, use `com.github.<Your_Username_Here>`
```

Comment:
```suggestion
        - If you don't know which Maven group to use, and you are planning to host the mod's source code on GitHub, use `io.github.<Your_Username_Here>`
```

Reply:
Oh right, I mixed up which one is good and which one is bad